adapters/Pangle/PangleAdapter/GADMAdapterPangleUtils.h

```diff
@@ -0,0 +1,13 @@
+//
+//  GADMAdapterPangleUtils.h
+//  PangleAdpter
+//
+//  Created by bytedance on 2021/12/16.
+//
+
+#import "GADMediationAdapterPangle.h"
+
+#define PangleLog(format, args...)    NSLog(@"PangleAdaper | "format,##args)
+
+NSError *_Nonnull GADMAdapterPangleErrorWithCodeAndDescription(GADPangleErrorCode code,
+                                                                 NSString *_Nonnull description);
```

Comment:
Can you also add another util:

```
void GADMAdapterPangleMutableSetAddObject(NSMutableSet *_Nullable set,
                                            NSObject *_Nonnull object) {
  if (object) {
    [set addObject:object];  // Allow pattern.
  }
}
```

And use it in all the other places that add object in mutable set?